scripts/generate-google3-docs/resource-reference/templates/bigqueryconnection_bigqueryconnectionconnection.tmpl

```diff
@@ -0,0 +1,54 @@
+{{template "headercomment.tmpl" .}}
+
+{% extends "config-connector/_base.html" %}
+
+{% block page_title %}{{ .Kind}}{% endblock %}
+{% block body %}
+{{template "alphadisclaimer.tmpl" .}}
+<table>
+<thead>
+<tr>
+<th><strong>Property</strong></th>
+<th><strong>Value</strong></th>
+</tr>
+</thead>
+<tbody>
+<tr>
+<td>{{"{{gcp_name_short}}"}} Service Name</td>
+<td>BigQueryConnection</td>
+</tr>
+<tr>
+<td>{{"{{gcp_name_short}}"}} Service Documentation</td>
+<td><a href="/bigquery/docs/">/bigquery/docs/</a></td>
+</tr>
+<tr>
+<td>{{"{{gcp_name_short}}"}} REST Resource Name</td>
+<td>bigqueryconnection/rest/v1/projects.locations.connections</td>
+</tr>
+<tr>
+<td>{{"{{gcp_name_short}}"}} Rest Resource Documentation</td>
+<td><a href="/bigquery/docs/reference/bigqueryconnection/rest/v1/projects.locations.connections">/bigquery/docs/reference/bigqueryconnection/rest/v1/projects.locations.connections</a></td>
```

Comment:
Should be `<td><a href="/bigquery/docs/reference/bigqueryconnection/rest/v1beta1/projects.locations.connections">/bigquery/docs/reference/bigqueryconnection/rest/v1beta1/projects.locations.connections</a></td>`. We are talking to the beta endpoint by default.